<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Task organizer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">*{margin: 0;
padding: 0;
background-color: #222222;
font: 9pt &quot;Roboto&quot;;
color: #ffffff;
width: 100%;
}

QListView {
    show-decoration-selected: 1;
	background-color: #2e3136;
    font: 12pt &quot;Roboto&quot;;
}

QListView::item:alternate {
    background-color: #40444b
}

QListView::item:selected {
    border: 1px solid #42d9c8;
}

QListView::item:selected:!active {
	background-color: rgb(36, 39, 43);
}
QListView::item:selected:active {
	background-color: rgba(116, 127, 139, 25);
}
QListView::item:selected:hover {
	background-color: rgba(116, 127, 139, 55);
}



QLineEdit#titleEdit, QPlainTextEdit#descriptionEdit, QDateEdit#deadlineEdit{
background-color: rgb(46, 50, 57);
border: none;
border-radius: 5px;

}
QLineEdit#titleEdit:focus, QPlainTextEdit#descriptionEdit:focus, QDateEdit#deadlineEdit:focus{
background-color: rgb(60, 65, 74);
border: 1px solid  rgb(77, 84, 95);

}
QCheckBox::indicator{
width: 50px;
height: 50px;
}
QCheckBox::indicator:unchecked{
image:url(:/images/images/unchecked-unhovered.png);
}
QCheckBox::indicator:unchecked:hover{
image: url(:/images/images/unchecked-hovered.png);
}
QCheckBox::indicator:unchecked:pressed{
image: url(:/images/images/unchecked-pressed.png);
}
QCheckBox::indicator:checked{
image: url(:/images/images/Checked-unhovered.png);
}
QCheckBox::indicator:checked:hover{
image:url(:/images/images/checked-pressed.png);
}
QCheckBox::indicator:checked:pressed{
image:url(:/images/images/checked-hovered.png);
}
.QPushButton{
width:100%;
height: 50px;
padding: 10px;
text-align:center;
border: none;
font: 15pt &quot;Roboto&quot;;
font-weight: bold;
margin: 0;
}
QPushButton#saveButton{
background-color:#5baa8d;
}
QPushButton#saveButton:hover{
background-color:#6fcfaa
}
QPushButton#saveButton:pressed{
background-color:#56a183
}
QPushButton#discardButton{
background-color:rgb(199, 93, 95)
}
QPushButton#discardButton:hover{
background-color:rgb(220, 103, 106)
}
QPushButton#discardButton:pressed{
background-color:rgb(179, 83, 87)
}
QPushButton#resetButton{
background-color: rgb(114, 175, 255);
}
QPushButton#resetButton:hover{
background-color: rgb(115, 206, 255);
}
QPushButton#resetButton:pressed{
	background-color: rgb(103, 144, 255);
}
QMenuBar{
background-color: rgb(60, 65, 74);
}
QMenuBar::item{
spacing: 5px;
background: transparent;
padding: 1px 4px;
border-radius: 4px;
}
QMenuBar::item:selected { 
    background: #58e2b2;
}
QMenuBar::item:pressed{
background: rgb(77, 198, 156);
}
QMenuBar::item:disabled{
background-color: rgb(38, 42, 49);
color: rgb(127, 127, 127);
}

QMenu {
    background-color: #2e3136;
    margin: 3px;
	
}

QMenu::item {
    padding: 2px 25px 2px 20px;
    border: 1px solid transparent; 
}

QMenu::item:selected {
    
	background-color: rgba(88, 226, 178, 150);
	border-color: #58e2b2;
}
QProgressBar#progressBar{
border: 1px solid rgb(81, 88, 97);
text-align: center;

}
QProgressBar#progressBar::chunk{
background-color: #58e2b2;
}





</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="listPage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="images.qrc">:/images/images/logo-small-01.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QListWidget" name="listWidget">
          <property name="tabKeyNavigation">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="titleEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="descriptionEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="deadlineLabel">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>64</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Deadline:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deadlineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2017</year>
              <month>5</month>
              <day>30</day>
             </datetime>
            </property>
            <property name="displayFormat">
             <string>dd-MM-yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2017</year>
              <month>5</month>
              <day>30</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="isDoneLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>64</horstretch>
              <verstretch>64</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Done:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="isDoneEdit">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="discardButton">
            <property name="text">
             <string>Discard</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>16</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuTasks">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Tasks</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
   </widget>
   <addaction name="menuTasks"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Add.png</normaloff>:/images/images/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Edit.png</normaloff>:/images/images/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Remove.png</normaloff>:/images/images/Remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>deadlineEdit</tabstop>
  <tabstop>isDoneEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>discardButton</tabstop>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
